packages/fx-core/src/common/basicLogin.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+"use strict";
+
+import { FxError, LoginStatus, ok, Result, TokenRequest } from "@microsoft/teamsfx-api";
+
+export abstract class BasicLogin {
```

Comment:
Any concern if moving this to api package?

Reply:
Thanks for the advice, move it to API package.